src/SFA.DAS.AdminService.Web/Services/Gateway/GatewayOverviewOrchestrator.cs

```diff
@@ -40,8 +38,87 @@ public async Task<RoatpGatewayApplicationViewModel> GetOverviewViewModel(GetAppl
                 return null;
             }
 
-            // Setting Application Data => TODO: To be stored in session.
-            var applicationData = new AssessorService.ApplyTypes.Roatp.Apply.Apply
+            var applicationData = GetApplicationData(application);
+
+            var viewmodel = new RoatpGatewayApplicationViewModel(applicationData);
+            viewmodel.Sequences = GetCoreGatewayApplicationViewModel();
+
+            var savedStatuses = await _applyApiClient.GetGatewayPageAnswers(request.ApplicationId);
+            if (savedStatuses != null && savedStatuses.Count.Equals(0))
```

Comment:
I would use !savedStatuses.Any() insteasd of savedStatuses.Count.Equals(0) as I think underlying sql is a little more efficient but it's not a big deal

Reply:
Because the logic checks for something in opposite to .Any() and now I simply negate it => !savedStatuses.Any()

Reply:
Done